android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java

```diff
@@ -370,6 +378,8 @@ public void extractEventData(RotationGestureHandler handler, WritableMap eventDa
       eventData.putDouble("anchorX", PixelUtil.toDIPFromPixel(handler.getAnchorX()));
       eventData.putDouble("anchorY", PixelUtil.toDIPFromPixel(handler.getAnchorY()));
       eventData.putDouble("velocity", handler.getVelocity());
+      eventData.putDouble("numberOfTouches", handler.getNumberOfPointers());
```

Comment:
Also, I'm curious why did you skip fling, tap and long press handlers when adding this new parameter? Looks like we can add it directly to the base class's `HandlerFactory.extractEventData` and call super in all subclasses (you can use `https://developer.android.com/reference/android/support/annotation/CallSuper.html` to ensure that it gets called)

Reply:
@kmagiera , 
Well, I wish to follow current iOS state and not to add this field where it was not before.
I found it also quite useles on fling as long as iOS requires exact number of pointers.
However, I will add it everywhere for consciousness